lib/api/src/conversions/vectors.rs

```diff
@@ -411,8 +411,8 @@ impl TryFrom<grpc::VectorsOutput> for VectorStructInternal {
 
 impl From<VectorInternal> for grpc::Vector {
     fn from(vector: VectorInternal) -> Self {
-        // ToDo: before deprecating `data`, `indices`, and `vectors_count`, ensure
-        // ToDo: that `vector` field is generated here.
+        // ToDo(v1.18): before deprecating `data`, `indices`, and `vectors_count`, ensure
```

Comment:
this specific part can be updated earlier, as it is only for internal cluster operations